server/ServerFSM.cpp

```diff
@@ -1321,6 +1370,11 @@ sc::result WaitingForTurnEndIdle::react(const SaveGameRequest& msg) {
     return transit<WaitingForSaveData>();
 }
 
+sc::result WaitingForTurnEndIdle::react(const Error& msg) {
```

Comment:
This appears to be overriding the containing state equivalent, but with nothing changed. Is it necessary?